### ISSUE TITLE
Last docs link refers to broken page (without version in URL)

### ISSUE THREAD

**Issue by shirokovnv** [hexpm/hexpm#1275]
I had similar issue with package: https://hex.pm/packages/phx_json_rpc

Main link forwards to the broken page.

Can you suggest, what's wrong with the package configuration ?

Or the problem sits in another place ?

Documentaion works fine locally.

_Originally posted by @shirokovnv in https://github.com/hexpm/hexpm/issues/600#issuecomment-2413822501_

Fixed with downgrading ex_doc version to 0.30.5

Still have issues with ex_doc 0.34.2

**Comment by ericmj** [comment]
What main link are you having issues it? Maybe you can show a screenshot of the issue on hexdocs and screenshot of what it looks like locally?

**Comment by shirokovnv** [comment]
I realized that I should start an issue in ex_docs repo. My apologies.

The problem I encountered was the following:

1) I published package in hexpm with docs (with ex_doc version: 0.34.2).

2) When I opened docs page URL, literally no sidebar (on the left) was shown.

3) In console there were an issue with some javascript variables.

I fixed an issue with downgrading ex_doc version to 0.30.5

Can't reproduce it for now, even with screenshot, because I republished package.

And locally everything works fine with both versions. Strange.

I will close the issue due to inconsistency. Thanks!
